### PR TITLE
chore: replace isort and flake8 with ruff

### PR DESCRIPTION
**Pull request by Fogapod**
Ruff runs at 8 times faster on this project than isort + flake8 while performing more checks (387ms v 47ms).

It seems like imports were not sorted properly and poetry config for dev dependencies was broken, i fixed that (though unsure about huge .lock file changes, i do not use poetry myself), as well as fixing a few other lints like module name `central-command -> central_command`, line lengths etc.

Overall this is pretty strict config with lots of rules, these can be relaxed if needed. Currently all checks pass.

Also bumped max line length from 88 to 121

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,18 +1,53 @@
...
+    "PTH",
+    # ruff
+    "RUF",
+]
+fixable = [
+    "I",
+]
+
+[tool.ruff.per-file-ignores]
+"settings.py" = ["E501"]
```
Comment (corp-0) [inline pyproject.toml]: What is this ignoring?

File: pyproject.toml [hunk 2]
```diff
@@ -1,18 +1,53 @@
 [tool.black]
-target-version = ['py38']
+target-version = ['py310']
```
Comment (corp-0) [inline pyproject.toml:2]: Maybe we could upgrade to python 3.11

File: pyproject.toml [hunk 3]
```diff
@@ -1,18 +1,53 @@
...
+
+[tool.ruff.per-file-ignores]
+"settings.py" = ["E501"]
+
+[tool.ruff.isort]
+combine-as-imports = true
+lines-between-types = 1
 
 [tool.poetry]
 name = "central-command"
```
Comment (corp-0) [inline pyproject.toml:58]: Did you forget to change the name here?

File: src/accounts/api/serializers.py [hunk 4]
```diff
@@ -46,7 +45,8 @@ def validate(self, data):
             )
         if not account.is_active:
             raise serializers.ValidationError(
-                "This account hasn't done the mail confirmation step or has been disabled."
+                "This account hasn't done the mail confirmation step "
+                "or has been disabled."
```
Comment (corp-0) [inline src/accounts/api/serializers.py]: Ewww, why did it do that

**Comment by corp-0** [comment]
Did you test if the app was running?

File: pyproject.toml (hunk 1 above)
Comment (Fogapod) [reply pyproject.toml]: complete list of rules: https://beta.ruff.rs/docs/rules/#pycodestyle-e-w
it's like too long, specifically for this line:
```py
        "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",
```
adding `# noqa` or splitting line both seemed bad. also previous flake8 config ignored this error anywhere, not just settings file

File: pyproject.toml (hunk 2 above)
Comment (Fogapod) [reply pyproject.toml:2]: not in this PR. 3.8 was outdated config. project is at 3.10 rn

File: pyproject.toml (hunk 3 above)
Comment (Fogapod) [reply pyproject.toml:58]: no, project name is independent of folder name

File: src/accounts/api/serializers.py (hunk 4 above)
Comment (Fogapod) [reply src/accounts/api/serializers.py]: 88 chars line limit. idk what's a better place to split string here

**Comment by Fogapod** [comment]
I did not test this because project has 0 docs on how to run it

File: src/accounts/api/serializers.py (hunk 4 above)
Comment (corp-0) [reply src/accounts/api/serializers.py]: Can you show me what it looks like in your IDE? I'm using my phone right now

**Comment by corp-0** [comment]
To run the app you do the usual. Make a .env file, the values on example.env should be perfectly fine for running locally. You then docker compose up the dev-compose file.

File: src/accounts/api/serializers.py (hunk 4 above)
Comment (Fogapod) [reply src/accounts/api/serializers.py]: the `-----` shows 88 chars limit, ruler on the right is 121 chars. Unwrapped it would be 91 chars

Comment (corp-0) [reply src/accounts/api/serializers.py]: Uhm, it doesn't look that ugly but yeah why 88 and not 121? I reckon people's monitors nowadays can afford it

Comment (Fogapod) [reply src/accounts/api/serializers.py]: i don't know. i took existing settings. it was 88 chars

Comment (corp-0) [reply src/accounts/api/serializers.py]: so can you make it 121?

Comment (Fogapod) [reply src/accounts/api/serializers.py]: i could but maybe in 2nd PR? it would force black to reformat a bunch of files

Comment (Fogapod) [reply src/accounts/api/serializers.py]: actually it's just a few files. give me a few minutes

Comment (Fogapod) [reply src/accounts/api/serializers.py]: done now

**Comment by corp-0** [comment]
tested locally and this works fine. Merging.

**Pull request merged**